src/htmx.d.ts

```diff
@@ -20,7 +20,7 @@ export function addClass(elt: Element, clazz: string, delay?: number): void;
  * @param path the URL path to make the AJAX
  * @param element the element to target (defaults to the **body**)
  */
-export function ajax(verb: string, path: string, element: Element): void;
+export function ajax(verb: string, path: string, element: Element): Promise<void>;
```

Comment:
Should we also update the documentation to mention the expected return type?

Reply:
The docs on the website linked above already state this return type. Do you mean adding a jsdoc `@returns` comment?

Reply:
Yeah, that's correct.